jdk/make/closed/autoconf/generated-configure.sh

```diff
@@ -4546,7 +4550,7 @@ VS_SDK_PLATFORM_NAME_2017=
 
 
 # Do not change or remove the following line, it is needed for consistency checks:
-DATE_WHEN_GENERATED=1604507962
+DATE_WHEN_GENERATED=1605209646
```

Comment:
Please include corresponding changes to `jdk/make/closed/autoconf/generated-configure.sh`.